doc/sphinx/source/conf.py

```diff
@@ -72,13 +72,14 @@
 
 autodoc_mock_imports = [
     'cartopy',
-    'cftime',
     'cf_units',
+    'cftime',
```

Comment:
`cftime` is a requested dependency (direct dependency) of `esmvalcore` - we shouldn't ask for it here if we ask for `esmvalcore` (which we do)

Reply:
It is also a direct dependency of ESMValTool, see here:

https://github.com/ESMValGroup/ESMValTool/blob/d75b479d42154676ce39fa8e06b97081aeb40c1f/esmvaltool/cmorizers/data/formatters/datasets/gpcc.py#L28

So maybe better leave it? Note that I didn't add this in this PR but rather changed the order of the list (so that is follows alphabetical order).

Reply:
aha, sorry, saw it in green and didn't see it in red - OK leave it as it is (for now) then, if it's imported directly we should keep it. Which reminds me - we should patch the meta recipe for the ESMValTool package on conda forge for the new PSY packages requested

Reply:
It should be fine to add these packages to conda forge when doing the next release, right? They are not needed for the current package. Will add an issue to the feedstock, though!

Reply:
yesh, an issue on the feedstock is enough for now :beer: